fetch-in-react-practice/client/src/components/Todos.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import React, { useEffect, useState } from 'react';
+import PageTitle from './PageTitle';
+import TodoList from './TodoList';
+import TodoForm from './TodoForm';
+
+const url = (path) => `${process.env.REACT_APP_BASE_URL}${path}`;
+
+export default function Todos() {
+  const [todos, setTodos] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+
+  useEffect(() => {
+    async function getRequest() {
+      try {
+        const response = await fetch(url`/api/todos`)
+        if (!response.ok) {
+          console.log('GET: Error');
```

Comment:
Logging here isn't a bad idea as long as you log something that will help you debug. As it is, you won't learn anything more than you will from the error message on the page. How about logging the response?